File: Projet_fait_a_la_main/src/main/java/mediathequeVirtuelle/model/Theme.java
package mediathequeVirtuelle.model;

public abstract class Theme {

	protected String label;
	
	protected int id;

	public Theme(String label, int id) {
		this.label = label;
		this.id = id;
	}

	public String getLabel() {
		return label;
	}

	public int getId() {
		return id;
	}

	public void setLabel(String label) {
		this.label = label;
	}

	public void setId(int id) {
		this.id = id;
	}

	@Override
	public String toString() {
		return "Theme [label=" + label + ", id=" + id + "]";
	}
	
}

File: angular/mediatheque_virtuelle/src/app/modele/theme.ts
import { EnumTheme } from "./enumTheme";

export class Theme {
    id: number;
    label: string;
    enumTheme: EnumTheme;

    constructor(id? : number, label?: string, enumTheme? : EnumTheme) {
        this.id=id;
        this.label=label;
        this.enumTheme=enumTheme;
    }
}